### ISSUE TITLE
[ENHANCEMENT-DEV] As MILES HARDIN, I need the idea submission form to account for the Strapi V.4 migration, so that I can seamlessly submit my ideas without any issues.

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#1272]
### Considerations:
Miles Hardin needs the idea submission form to account for the Strapi V.4 migration so that he can submit his ideas without any issues. 

### Acceptance criteria:
- The idea submission form should be updated to account for the Strapi V.4 migration.
- The schema in Strapi-V4 ideacard should be changed to conform to the stagging schema (https://docs.google.com/document/d/1NaJyGubyFhOAYwh0FMp4cp2B3-sPoPS1Hrf1usvW9FY/edit?pli=1).
- The code migration script and database migration script should be used to assist in the migration process.
- The updated form should be able to handle any breaking changes that come with the migration.
- The updated form should be able to handle any custom code or modifications made to the existing code.
- The updated form should be tested to ensure that it is working seamlessly on all devices and [target browsers ](https://docs.google.com/spreadsheets/d/1upYP1mvHkU3sRBg4gCeGG4P63tO5-whn6VKLMlr6qYI/edit?usp=sharing): Chrome, Microsoft Edge, Safari, Firefox and Opera.

----------------------------------------------------
### Sub-tasks:
- [ ] Back end Development
- [ ] QA

**Comment by JulieMass** [comment]
Hi @alicefate, @dbradham mentioned to me today that you discovered we would need to account for the Strapi V4 migration for the idea submission so I created this ticket. 

I would need both your help with the following:

1. Can you review the ticket description and acceptance criteria and let me know if it is OK? Is there something I missed or should be changed? 
2. Is this something you’ll be working on this sprint Iris or following ones? 

Thank you so much both in advance!

**Comment by alicefate** [comment]
acceptance criteria:
The schema in Strapi-V4 ideacard should be changed to conform to the stagging schema (https://docs.google.com/document/d/1NaJyGubyFhOAYwh0FMp4cp2B3-sPoPS1Hrf1usvW9FY/edit?pli=1).

**Comment by JulieMass** [comment]
> acceptance criteria: The schema in Strapi-V4 ideacard should be changed to conform to the stagging schema (https://docs.google.com/document/d/1NaJyGubyFhOAYwh0FMp4cp2B3-sPoPS1Hrf1usvW9FY/edit?pli=1).

Excellent! Thank you Iris. I included it in the acceptance criteria: 
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/104226366/93699cf4-685e-49c3-84ec-c064cc6ed2e1)

Are you considering taking this ticket in the current sprint or in future sprints? I'm asking because if you plan to take it in this sprint, we should proceed with poker planning for everyone. However, if you intend to take this ticket in future sprints, we can hold off on estimating it. Please let me know your preference.

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/1272?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @katehirschman @MariliaDP @RETprojects

**Comment by dbradham** [comment]
@JulieMass This looks good from my testing.

**Comment by JulieMass** [comment]
(CC. @chadcrotchett )

Update: After Dave's feedback about the issue with duplicated named ideas, I've tried again today and I successfully submitted our idea for staging! I agree with him also that there's a pop-up during submission that might be confusing but we've met the acceptance criteria for this ticket, so we are able to close it.

Thank you so much @RETprojects  and @dbradham  for your hard work! 

<img width="1332" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMmViQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c420f0055287d6352547b38b67e263928694bb7c/image.png" alt="image.png" />

<img width="1341" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMnFiQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cbe505567d530df4ca2488a70bb0f8c0c8eea1aa/image.png" alt="image.png" />
